zbus_xmlgen/tests/data/sample_object0.rs

```diff
@@ -1,6 +1,9 @@
 #[dbus_proxy(interface = "com.example.SampleInterface0", assume_defaults = true)]
 trait SampleInterface0 {
 
+    /// BarplexSig method
+    fn barplex_sig(&self, rule: &(&[i32], i32, std::collections::HashMap<&str, &str>, i32, &[i32], i32, &[&str], i32, bool)) -> zbus::Result<Vec<(String, zbus::zvariant::OwnedObjectPath)>>;
```

Comment:
Too long a line. The lint part of the CI will fail on this. Please run `cargo +nightly fmt` on you machine to ensure that all code is correctly-formatted.

Reply:
Actually,[ I was wrong](https://github.com/dbus2/zbus/actions/runs/6779306008/job/18441228520?pr=506). :laughing: This file is not part of the build so `cargo fmt` will not see it. You need to run `rustfmt zbus_xmlgen/tests/data/sample_object0.rs` manually. It would be great if we could somehow check the formatting for this file in the CI.

Reply:
Hi, checking the comments now!

This test file I actually had to disable the auto formatter from my editor :grimacing:, because the test was failing with the generated code and I didn't wanted to tweak with the test (only to add a new case).

Do you think it is necessary to format it? It would be necessary to adapt the test macro I think.

Reply:
> This test file I actually had to disable the auto formatter from my editor 😬, because the test was failing with the generated code and I didn't wanted to tweak with the test (only to add a new case).

You can always have other changes in the same branch/PR, just commit them separately (if you use commandline, `git add -p` is your friend).

> Do you think it is necessary to format it? It would be necessary to adapt the test macro I think.

Right. That would be great. However, you can do that in a separate PR afterwards.